pyqtorch/apply.py

```diff
@@ -57,3 +59,35 @@ def apply_operator(
         map(lambda e: "".join(list(e)), [operator_dims, in_state_dims, out_state_dims])
     )
     return einsum(f"{operator_dims},{in_state_dims}->{out_state_dims}", operator, state)
+
+
+def apply_op_op(operator_1: Tensor, operator_2: Tensor, target: int) -> Tensor:
```

Comment:
Maybe `apply_op_to_op` ? Merely a suggestion.

Reply:
It makes sense to change the name. 
Perhaps `opmul`, akin to `torch.matmul()`, would be better as it is shorter and more explicit. 
However, if we aim to maintain the same syntax as the `apply_operator()` function, then `apply_op_to_op` would be more suitable. 
Which name should we stick with?

Reply:
Hm, I would be inclined to use `op_mul` but we can decide together with @dominikandreasseitz